mlir/test/lit.cfg.py

```diff
@@ -0,0 +1,77 @@
+# -*- Python -*-
+
+import os
+import platform
+import re
+import subprocess
+import tempfile
+
+import lit.formats
+import lit.util
+
+from lit.llvm import llvm_config
+from lit.llvm.subst import ToolSubst
+from lit.llvm.subst import FindTool
+
+# Configuration file for the 'lit' test runner.
+
+# name: The name of this test suite.
+config.name = "hugr-mlir"
+
+config.test_format = lit.formats.ShTest(not llvm_config.use_lit_shell)
```

Comment:
This line could use a comment too!

Reply:
hmm, "cargo culted from standalone project".